src/main/java/org/apache/pirk/responder/wideskies/storm/EncColMultBolt.java

```diff
@@ -103,13 +101,13 @@ public void execute(Tuple tuple)
       // Data tuple received. Do column multiplication.
 
       long colIndex = tuple.getLongByField(StormConstants.COLUMN_INDEX_ERC_FIELD);
-      colVal1 = (BigInteger) tuple.getValueByField(StormConstants.ENCRYPTED_VALUE_FIELD);
+      BigInteger colVal1 = (BigInteger) tuple.getValueByField(StormConstants.ENCRYPTED_VALUE_FIELD);
 
       logger.debug("Received tuple in ECM, multiplying {} to col {}", colVal1, colIndex);
 
       if (resultsMap.containsKey(colIndex))
       {
-        colMult = colVal1.multiply(resultsMap.get(colIndex));
+        BigInteger colMult = colVal1.multiply(resultsMap.get(colIndex));
         resultsMap.put(colIndex, colMult.mod(nSquared));
```

Comment:
I don't think it changes the initialization, even earlier the initialization was happening in the execute().